### PR TITLE
Fix change Authority issues

### PR DESCRIPTION
**Pull request by lucatume**
Ticket: https://central.tri.be/issues/76534

Builds on: https://github.com/moderntribe/tribe-common/pull/358

### CODE REVIEW

**Comment by zbtirrell** [comment]
It doesn't feel like this is respecting what is happening above - https://github.com/moderntribe/the-events-calendar/pull/1353/files#diff-f8677ebdc85d2610cfcd992b86b08790L360

File: src/Tribe/Aggregator/Event.php [hunk 1]
```diff
@@ -349,27 +353,27 @@ public static function preserve_changed_fields( $data ) {
 				continue;
 			}
 
-			// if we don't have a field to reset to, let's unset the event meta field
-			if ( ! isset( $post_meta[ $field ] ) ) {
-				unset( $data[ $field ] );
```
Comment (bordoni) [inline src/Tribe/Aggregator/Event.php]: Don't we need to still unset the data field? I mean from what I remember without it would break.

Comment (lucatume) [inline src/Tribe/Aggregator/Event.php]: we set it to an empty string later in the code if the original is empty

Comment (lucatume) [inline src/Tribe/Aggregator/Event.php]: I mean here:  https://github.com/moderntribe/the-events-calendar/pull/1353/files#diff-f8677ebdc85d2610cfcd992b86b08790R364

Comment (bordoni) [inline src/Tribe/Aggregator/Event.php]: Why not Unset it? I mean avoid to trigger an update.

Comment (lucatume) [inline src/Tribe/Aggregator/Event.php]: Ok, latest version has an explicit unset

**Pull request merged**